src/stores/room-list/ListLayout.ts

```diff
@@ -89,11 +89,12 @@ export class ListLayout {
         return 5 + RESIZER_BOX_FACTOR;
     }
 
-    public setVisibleTilesWithin(diff: number, maxPossible: number) {
+    public setVisibleTilesWithin(newVal: number, maxPossible: number) {
+        maxPossible = maxPossible + RESIZER_BOX_FACTOR;
         if (this.visibleTiles > maxPossible) {
```

Comment:
Shouldn't this condition be testing `newVal`, not the existing value...? I am probably missing something...

Reply:
err... possibly. I'll run through the cases in a bit to double check and report my findings.

Reply:
yes, this is a thing we're supposed to do: the old code relied on a different behaviour happening and now instead of providing a diff to the function we're providing a new height, so the first part of the ladder needs updating.